db/patterns/twitter_button.eno

```diff
@@ -1,8 +1,8 @@
-name: Twitter Button
+name: X Button
```

Comment:
Maybe "Tweet Button" would be the best name? From what I see, this is how they refer to it. We could also update the website_url to point to:

https://developer.twitter.com/en/docs/twitter-for-websites/tweet-button/overview

Side-note: The filters below also seem to be dead, but upgrading them is a topic on its own. Thus, I split it out to https://github.com/ghostery/trackerdb/issues/105.